packs/spell-effects/spell-effect-dragon-claws.json

```diff
@@ -1,10 +1,10 @@
 {
     "_id": "7zy4W2RXQiMEr6cp",
     "img": "systems/pf2e/icons/spells/dragon-claws.webp",
-    "name": "Spell Effect: Flurry of Claws",
+    "name": "Spell Effect: Dragon Claws",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Flurry of Claws]</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Flurry of Claws]{Dragon Claws}</p>\n<p>Vicious claws grow from your fingers. They are finesse unarmed attacks that deal slashing damage and extra damage of a type determined by the dragon in your bloodline. You gain resistance to the same damage type.</p>"
```

Comment:
```suggestion
            "value": "<p>Granted by <em>Dragon Claws</em></p>\n<p>Vicious claws grow from your fingers. They are finesse unarmed attacks that deal slashing damage and extra damage of a type determined by the dragon in your bloodline. You gain resistance to the same damage type.</p>"
```

Reply:
Since that spell doesn't really exist in that form any longer, maybe we shouldn't link to the remaster one?